spec/features/emails_spec.rb

```diff
@@ -423,52 +422,43 @@
       expect(email).to have_body_text(investment.unfeasibility_explanation)
     end
 
-    scenario "Selected investment" do
+    scenario 'Selected investment' do
```

Comment:
We haven't yet had the discussion about the usage of double and single quotes to enable/disable the rubocop rule explicitly. For now please just follow existing conventions in files instead of changing consistency. This applies to any "" for '' replacement (in both ways)